roles/eda/defaults/main.yml

```diff
@@ -169,5 +169,8 @@ admin_password_secret: ''
 # Disable UI container's nginx ipv6 listener
 ipv6_disabled: false
 
+# Define which address to bind in the api, depending on the enabled IP version
+api_bind_address: "{{ (ipv6_disabled | bool) | ternary('0.0.0.0', '[::]') }}"
```

Comment:
This shouldn't go to `defaults` but `vars` directory as this isn't something that users can customize at CR level

https://github.com/ansible/eda-server-operator/blob/main/roles/eda/vars/main.yml